ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -53,6 +53,82 @@ tests:
   container:
     clone: true
     from: golangci-lint
+- as: check-commits-count
+  commands: ci/prow/check-commits-count
+  container:
+    from: src
+- as: check-api-changes
+  commands: |
+    unset GOFLAGS
+    ci/prow/check-api-changes
+  container:
+    from: src
```

Comment:
Those Makefile targets requires additional tools such as `kubectl`, `operator-sdk` and more.

`src` isn't the right image to use here.